docs/registry/README.md

```diff
@@ -0,0 +1,193 @@
+# Registry Proposal
+
+## Objective
+
+Design an **initial** version of the Registry that can support discoverability of 
+the different event types that can be consumed from the eventing mesh.
+
+## Requirements
+
+Our design revolves around the following core requirements:
+
+1. We should have a Registry per namespace to enforce isolation.
+2. The Registry should contain the event types that can be consumed from
+the eventing mesh. If an event type is not ready for consumption, we should explicitly indicate so.
```

Comment:
What would cause an EventType to "not [be] ready for consumption"?

Reply:
E.g., that the broker doesn't exist or the broker is not Ready

Reply:
So EventType's ready is just a proxy for its Broker's ready?

Reply:
As of now, yes. Broker existence and broker readiness.